improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -327,6 +395,12 @@ def set_up_variable_cube(
     )
     cube.rename(name)
 
+    # add bounds on spatial coordinates
+    if ypoints > 1:
+        cube.coord(axis="y").guess_bounds()
+    if xpoints > 1:
+        cube.coord(axis="x").guess_bounds()
```

Comment:
Can this be done as part of the spatial coordinate creation, in `construct_xy_coords`?

Reply:
Updated [here](https://github.com/metoppv/improver/pull/1309/commits/1d7f21003ca70fbe79b8c8943e280bc37f411b98#diff-ea38258437543d699495d248fce61e29L102).